target/linux/kirkwood/patches-5.4/112-readynas_duo_v2.patch

```diff
@@ -0,0 +1,72 @@
+--- a/arch/arm/boot/dts/kirkwood-netgear_readynas_duo_v2.dts
++++ b/arch/arm/boot/dts/kirkwood-netgear_readynas_duo_v2.dts
+@@ -19,6 +19,13 @@
+ 		reg = <0x00000000 0x10000000>;
+ 	};
+ 
++	aliases {
++		led-boot = &led_power;
++		led-failsafe = &led_power;
++		led-running = &led_power;
++		led-upgrade = &led_power;
++	};
++
+ 	chosen {
+ 		bootargs = "console=ttyS0,115200n8 earlyprintk";
+ 		stdout-path = &uart0;
+@@ -115,7 +122,7 @@
+ 			      &pmx_led_blue_backup >;
+ 		pinctrl-names = "default";
+ 
+-		power_led {
++		led_power: power_led {
+ 			label = "status:blue:power_led";
```

Comment:
Is there a specific reason for these complicated (and non-standard) LED labels? Why not just

blue:power
blue:disk1
blue:disk2

?

Reply:
Err, please ignore what I just wrote, I'm too stupid to understand a patch when I see it (and the fact that this is actually upstream). Sorry.